spec/features/user_edits_email_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'user edits email', %Q{
+  As an authenticated user, I want to edit my email, so that I can continue using my account when I get a new email address.
+  Acceptance Criteria:
+  * [X] - I can update my email address.
+  * [X] - I see an error if my new email address is invalid.
+  * [ ] - ~~I must confirm my email address.~~
+  * [ ] - ~~If I don’t confirm my email address within 24 hours, it will not be saved.~~
+  } do
+
+    let(:user) do
+      FactoryGirl.create(:user)
+    end
+
+    scenario 'user edits optional information' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.